server/user-bootstrap/index.js

```diff
@@ -25,6 +25,7 @@ const debug = debugFactory( 'calypso:bootstrap' ),
 	apiQuery = {
 		meta: 'flags',
 		abtests: getActiveTestNames( { appendDatestamp: true, asCSV: true } ),
+		active_tests: config.isEnabled( 'ive/me' ),
```

Comment:
Right now, I've set this to just send `true`, but it would be better to send an actual configuration string (such as `product=true&abtest_cookie=something` or something like that.